src/tests/utils.py

```diff
@@ -249,7 +249,7 @@ def assert_test(test_func, all_kwargs, dev_type=None,
         writer.write_api(method_name, all_kwargs, overwrite)
         return True
     if writer.existing_api(method_name) is False:
-        logger.debug("No existing, API data for %s %s %s", mod, cls_name, method_name)
+        logger.error("No existing, API data for %s %s %s", mod, cls_name, method_name)
```

Comment:
Assuming this is stating we don't have code for the API.   Could this be a warning vs error?